### ISSUE TITLE
Bug: [calcite-dropdown] Tabbing is broken between instances of Dropdown

### ISSUE THREAD

**Issue by macandcheese** [Esri/calcite-components#778]
### Summary
Something in beta.32 broke the ability to tab between instances of calcite-dropdown

### Actual Behavior
Tab position is lost after closing or tabbing through a dropdown instance (do not need to open the dropdown menu), preventing advancing to the next tab-able item in the page flow.

### Expected Behavior
You should be able to tab through the page after interacting with a calcite-dropdown

### Reproduction Steps

1. Try to tab through the dropdown items on the local dev demo dropdown page. Last working release was `beta.31`
